.github/workflows/nv-torch113-p40.yml

```diff
@@ -1,7 +1,8 @@
-name: nv-torch110-p40
```

Comment:
Update this to test 1.11 as this is the latest supported by transformers/accelerate.

Reply:
Done.